### DISCUSSION TITLE
MISP to ElastAlert

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#5004] [Unsupported Versions] [answered]

**Comment by weslambert** [comment, marked as answer]
I actually looked into this same thing (from prior to the edit -- sigmai) a while back. 😄  Currently, there is no ability to use a custom rule repo with Playbook.  Once that is in place, you could have Playbook pull in the rules and convert/map them as needed. Otherwise, there is nothing stopping you from performing your own conversion and having the rules placed under the Elastalert rule directory, but it would be cleaner and more supported once the ability to use a custom rule repo with Playbook is available. You may also consider using the following in the time being (not quite the same (for Zeek/Suricata specifically), but could be beneficial): https://github.com/weslambert/securityonion-misp

**Reply by Jackson-Pollock** [reply]
Thank you!
